src/windows/winjector-helper-core.c

```diff
@@ -380,18 +382,27 @@ initialize_remote_worker_context (RemoteWorkerContext * rwc,
   /* } */
   gum_x86_writer_put_label (&cw, skip_unload_label);
 
+  /* set return 0 */
+  gum_x86_writer_put_xor_reg_reg(&cw, GUM_REG_EAX, GUM_REG_EAX);
```

Comment:
Inconsistent style as per:
https://github.com/frida/frida/blob/master/CONTRIBUTING.md#rule-12

(Here and elsewhere, including macro invocations.)

Reply:
done.